src/cwStationRenamer.h

```diff
@@ -43,11 +37,8 @@ class cwStationRenamer
     QHash<QString, QString> ValidToInvalidStations;
     QSet<QString> ValidStations;
 
-    QPointer<cwCave> Cave;
-
-    void renameInvalidStationsInChunk(cwSurveyChunk* chunk,
-                                      int stationIndex,
-                                      const QRegExp& removeInvalidCharsRegex);
+    static const QRegExp validUcaseRx;
```

Comment:
For consistency naming across class make validUCaseRx and invalidCharRx has the first character upper case. For example ValidUCaseRx and InvalidCharRx. Your can easily refactor rename by doing ctrl-R in Qt-Creator. Also it's helpful to not abbreviate. I wasn't completely clear what ValidUcase did, but ValidUpperCase is slightly more helpful.